src/main/webapp/scripts/toggle.js

```diff
@@ -55,6 +67,7 @@ async function loadKeywordSection(keyword) {
   const keywordObj = buildKeywordObj(objs[0], objs[1], keyword);
   const template = await KEYWORD_TEMPLATE_PROMISE;
   renderKeyword(template, keywordObj);
+  return true;
```

Comment:
What is the purpose of returning true here.

Reply:
Keeping track of the functions that haven been completed. Without a return value is not possible to wait until the promise resolves.